ui/components/Notification/MesheryEventViewer.js

```diff
@@ -0,0 +1,20 @@
+import MesherySnackbarWrapper from "./MesherySnackbarWrapper"
+
+
+//Component for the  event(notification) snackbar 
```

Comment:
Let us stay consistent with the way we add comments. For any declarations, we shall use multiline comments( `/* */` ) and leverage JsDoc to give additional type information.